cgp/ea/mu_plus_lambda.py

```diff
@@ -71,7 +71,10 @@ def initialize_fitness_parents(
         pop._parents = self._compute_fitness(pop.parents, objective)
 
     def step(
-        self, pop: Population, objective: Callable[[IndividualBase], IndividualBase]
+        self,
+        pop: Population,
+        objective: Callable[[IndividualBase], IndividualBase],
+        reorder_genome: bool = False,
```

Comment:
```suggestion
```
please make `_reorder_genome` a member of the `MuPlusLambda` class which is an argument in `MuPlusLambda.__init__` and defaults to `True`

Reply:
What should I do with the other examples and the tests? Add an additional argument `genome_reorder = False` to `ea_params`?